### PR TITLE
Set user-agent in pysnyk client

### PR DESCRIPTION
**Pull request by maxjeffos**
Set the user-agent in the pysnyk client so that we know that API calls are coming from this tool. Had to bump to the latest version of pysnyk. Added a test to make sure that the __version__ in the module init file matches that in the pyproject.toml

### CODE REVIEW

File: snyk_threadfix/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__version__ = '0.0.3'
```
Comment (garethr) [inline snyk_threadfix/__init__.py]: Rather that hardcoding this in two places, I'd use the https://github.com/rominf/poetry-version library.

File: snyk_threadfix/main.py [hunk 2]
```diff
@@ -252,7 +253,8 @@ def main(args):
     if not token_is_valid:
         raise SnykTokenInvalidError('invalid token')
 
-    client = snyk.SnykClient(snyk_token)
+    user_agent_string = 'snyk-threadfix/%s' % __version__
```
Comment (garethr) [inline snyk_threadfix/main.py:256]: I'd definitely recommend starting to use `black` and `isort`, consistency of formatting is super useful when it comes to maintenance over time.

File: tests/test_main.py [hunk 3]
```diff
@@ -574,3 +577,20 @@ def test_create_finding_data_regular_project_from_cli_project_with_custom_name()
...
     assert tf_finding['mappings'][1]['primary'] is True
+
+
+def test_module_version_matches_pyproject_version():
+    """So we don't update the version in one place only and not both"""
+    version_from_package_init = __version__
+
+    # this is so that the test finds the pyproject.toml file when run from the command line or from within Pycharm
+    this_directory = os.path.dirname(os.path.realpath(__file__))
+    pyproject_toml_path = this_directory + '/../pyproject.toml'
```
Comment (garethr) [inline tests/test_main.py]: This assumes you're on a system where forward slashes are the path separator. Best to use `os.path.join` to make things like this cross platform.

File: snyk_threadfix/main.py (hunk 2 above)
Comment (maxjeffos) [reply snyk_threadfix/main.py:256]: Yes! I will. I have it on the board as BDE-16: https://snyksec.atlassian.net/browse/BDE-16

File: snyk_threadfix/__init__.py (hunk 1 above)
Comment (maxjeffos) [reply snyk_threadfix/__init__.py]: Nice! Thanks

File: tests/test_main.py (hunk 3 above)
Comment (maxjeffos) [reply tests/test_main.py]: fixed

**Pull request merged**